#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_fix_infinite_loop.dpatch by Andrew Lee <andrew@linux.org.tw>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: solve the problem on runs configure in infinite loop.

@DPATCH@
diff -urNad lxrandr-0.1.1~/configure.in lxrandr-0.1.1/configure.in
--- lxrandr-0.1.1~/configure.in	2009-06-25 04:59:58.000000000 +0800
+++ lxrandr-0.1.1/configure.in	2009-07-15 22:09:42.453865793 +0800
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC
